CMakeLists.txt

```diff
@@ -235,6 +235,12 @@ If you already downloaded the source, you could try to re-configure this project
 endif()
 message(STATUS "SRC_DIR: ${SRC_DIR}")
 
+# Apply patches
+option(PYTHON_APPLY_PATCHES "Apply patches" ON)
+if(PYTHON_APPLY_PATCHES)
+  include(cmake/PythonApplyPatches.cmake)
+endif()
```

Comment:
Because `PythonApplyPatches.cmake` uses `PY_VERSION`, this block should come after the block below that extracts the version from the source.